### PR TITLE
Add OTP ViewController, Storyboard and UITextFieldDelegate extension.

### PR DESCRIPTION
**Pull request by 05Waleed**
## Description 

1) Added shadow func 
2) UITextFieldDelegate extension 

## ScreenShots/gifs

https://github.com/haroonmaqsoodch/ShopGo/assets/139551937/a39e6ad9-b6a2-4b8f-9786-25cce51168e8

### CODE REVIEW

**Comment by zuhair5** [comment]
Everything seems alright to me.

File: ShopGo/OTPViewController+Storyboard/OTPViewController.swift
Comment (zuhair5) [inline ShopGo/OTPViewController+Storyboard/OTPViewController.swift]: Try to make the extension in a separate swift file and call the relevant methods in the view did load to make this view controller page as clean as possible, apart from that everything seems good.

File: ShopGo/OTPViewController+Storyboard/OTPViewController.swift
Comment (haroonmaqsoodch) [inline ShopGo/OTPViewController+Storyboard/OTPViewController.swift]: break should move to the next line

File: ShopGo/OTPViewController+Storyboard/OTPViewController.swift
Comment (haroonmaqsoodch) [inline ShopGo/OTPViewController+Storyboard/OTPViewController.swift]: Delay of 0.1 and 0.3 is being used multiple times, create a constants on the top and define these values there then pass the constant here instead of passing the values again and again

File: ShopGo/OTPViewController+Storyboard/OTPViewController.swift
Comment (haroonmaqsoodch) [inline ShopGo/OTPViewController+Storyboard/OTPViewController.swift]: Do not use force unwrapping

**Review by haroonmaqsoodch — COMMENTED** [review]
Also the field is taking multiple values, limit it to take only one digit

Simulator Screenshot - iPhone 14 Pro - 2023-07-22 at 15 08 00

**Review by haroonmaqsoodch — APPROVED** [review]
LGTM! Wait for @zuhair5's approval then he will merge the PR

**Pull request merged**
